### PR TITLE
Create CrossChainBalance.md

### PR DESCRIPTION
**Pull request by themandalore**

### CODE REVIEW

File: types/CrossChainBalance.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+The `CrossChainBalance` query has two parameters, which specifies the requested data.
+
+1. **chainId** (string): ID of the proposal.
+
+The `chainId` should be a valid EVM chain ID.
+
+1. **contractAddress** (string): ID of the proposal.
+
+The `contractAddress` should be a valid token contract on the given EVM chain.
+
```
Comment (tkernell) [inline types/CrossChainBalance.md:20]: add `blockNumber`

**Comment by tkernell** [comment]
this looks good

File: types/CrossChainBalance.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+## Query Description
+
+This query returns the merke root for a given merkle tree of a token's balances. Reporters will create a Merkle Tree of all addresses and their balances at a certain block and then upload the bytes32 merkle root.
+
+## Query Parameters
+
+The `CrossChainBalance` query has two parameters, which specifies the requested data.
+
+1. **chainId** (string): ID of the proposal.
```
Comment (oraclown) [inline types/CrossChainBalance.md:13]: `uint256`? Remove "ID of the proposal"

File: types/CrossChainBalance.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+The `chainId` should be a valid EVM chain ID.
+
+1. **contractAddress** (string): ID of the proposal.
```
Comment (oraclown) [inline types/CrossChainBalance.md:17]: `address`. Remove "ID of the proposal"

File: types/CrossChainBalance.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+- It is the reporters responsibility to calculate out the correct merkle root.  
+
+
+## Example
+
+In the example query:  
+
+```
+abi.encode("CrossChainBalance", abi.encode(1,0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0,15998590))
```
Comment (oraclown) [inline types/CrossChainBalance.md:88]: to be more clear:
```c
queryData = abi.encode("CrossChainBalance", abi.encode(1,0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0,15998590))
```
?

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks @themandalore

File: types/CrossChainBalance.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+## Query Description
+
+This query returns the merke root for a given merkle tree of a token's balances. Reporters will create a Merkle Tree of all addresses and their balances at a certain block and then upload the bytes32 merkle root.
+
+## Query Parameters
+
+The `CrossChainBalance` query has three parameters, which specifies the requested data.
+
+1. **chainId** (string):
```
Comment (oraclown) [inline types/CrossChainBalance.md:13]: change type from string to uint256

File: types/CrossChainBalance.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+The `chainId` should be a valid EVM chain ID.
+
+2. **contractAddress** (string):
```
Comment (oraclown) [inline types/CrossChainBalance.md:17]: change type from string to address

File: types/CrossChainBalance.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+The `contractAddress` should be a valid token contract on the given EVM chain.
+
+3. **blockNumber** (string):
```
Comment (oraclown) [inline types/CrossChainBalance.md:21]: change type from string to uint256

**Review by oraclown — CHANGES_REQUESTED** [review]
Minor type fixes needed

File: types/CrossChainBalance.md (hunk 7 above)
Comment (themandalore) [reply types/CrossChainBalance.md]: done

**Pull request merged**